### DISCUSSION TITLE
Loco positioning

### DISCUSSION THREAD

**Discussion by ivankaDov** [bitcraze/discussions#1389] [Q&A]
Hi! How I can get a position of the crazyflie using Loco positioning system to create an app layer? What functions or libraries should I use? Is it possible on C? Furhtermore, what functions I can use on Python to get the current crazyflie position using Loco Positioning?

**Comment by gemenerik** [comment]
Positioning is managed through the Kalman filter, so you don't need to use any Loco Positioning System (LPS)-specific functions directly. The Kalman filter integrates data from the LPS and other sensors to estimate the Crazyflie's position.

**Reply by gemenerik** [reply]
So to clarify, you can use the default methods of getting the position, whether it's from the app layer or the Python library. Hope that helps you get started!

**Reply by ivankaDov** [reply]
Thank you! I will try.
